pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -201,29 +206,41 @@ func createTargetConfig(ctx context.Context, c TargetConfigController, recorder
 	return false, nil
 }
 
-func manageKubeAPIServerConfig(client coreclientv1.ConfigMapsGetter, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
+func manageKubeAPIServerConfig(client coreclientv1.ConfigMapsGetter, infrastructureLister configlisterv1.InfrastructureLister, recorder events.Recorder, operatorSpec *operatorv1.StaticPodOperatorSpec) (*corev1.ConfigMap, bool, error) {
+	requiredConfigMap, err := manageKubeAPIServerConfigInternal(operatorSpec, infrastructureLister)
+	if err != nil {
+		return nil, false, err
+	}
+	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
+}
+
+func manageKubeAPIServerConfigInternal(operatorSpec *operatorv1.StaticPodOperatorSpec, infrastructureLister configlisterv1.InfrastructureLister) (*corev1.ConfigMap, error) {
```

Comment:
this is not `manage*`, it's `computeKubeApiserverConfigMap` or something like that.